### PR TITLE
Add AG type and RC Smoothing type info to Header

### PR DESCRIPTION
**Pull request by McGiverGim**
This add Anti Gravity type and RC Smoothing type info to the header page. I think is better to not merge until the RC Smoothing part is finished in the configurator, in this way I can try to maintain this more similar to configurator (name, order of fields, etc.)

### CODE REVIEW

**Comment by McGiverGim** [comment]
This is affected by https://github.com/betaflight/betaflight/issues/6490 so maybe is better to wait to add this other fields too.

**Comment by McGiverGim** [comment]
@etracer65 two questions about this: 
- I have seen that the Blackbox have a header `rc_smoothing_debug_axis` do you think that is useful for the user to show this value in the Header page of the Blackbox Viewer?
- One thing that is missing, and maybe is useful is the axes for the rc smoothing/interpolation. They are not printed in the Blackbox and this seems useful.

By the rest the PR is ready. If the debug axis is useful I can add it. If the axes are added to the firmware I will add them too.

**Comment by etracer65** [comment]
@McGiverGim It's probably not critical to add the `rc_smoothing_debug_axis` but it wouldn't hurt.  Also if the goal is to eliminate the unknown fields then I guess we should.  The field is useful when looking at logs submitted by users otherwise we have know idea which rcCommand axis to compare against so that's why it's in there.

Adding the interpolation channels is also a great idea and I'm surprised it wasn't already there.  I created a PR https://github.com/betaflight/betaflight/pull/6527 to add the field to the header.  The PR has the details of the field name and values for your development.

**Comment by McGiverGim** [comment]
Perfect, I will wait to see if it is tagged for 3.5 or 4.0 to finish this PR. Thanks!

**Comment by McGiverGim** [comment]
Tagged for 4.0, so added only by the moment the debug axis:

image

Ready to be merged :)

**Comment by etracer65** [comment]
Well I marked it 4.0 since it didn't seem appropriate to go into 3.5 at this point.  But you can go ahead and code it if you want and just get it done.  I can give you a log with it contained if you need.

**Comment by McGiverGim** [comment]
No, I prefer to wait until merged, because I need to detect version and maybe reorder the elements in the screen.

**Comment by McGiverGim** [comment]
Updated, I forgot the rc_smoothing_rx_average because I suffer the bug with the sdcard not writing all the headers :P

**Comment by mikeller** [comment]
@McGiverGim: Is this ready to go now?

**Comment by McGiverGim** [comment]
Yes, it is ready :)

File: js/header_dialog.js [hunk 1]
```diff
@@ -539,8 +547,28 @@ function HeaderDialog(dialog, onSave) {
 		setParameter('gyro_lowpass_hz'			,sysConfig.gyro_lowpass_hz,0);
 		setParameter('gyro_lowpass2_hz'         ,sysConfig.gyro_lowpass2_hz,0);
 
-    	renderSelect('rc_interpolation'		    ,sysConfig.rc_interpolation, RC_INTERPOLATION);
-        setParameter('rc_interpolation_interval',sysConfig.rc_interpolation_interval,0);
+        renderSelect('rc_smoothing_type'          ,sysConfig.rc_smoothing_type, RC_SMOOTHING_TYPE);
+    	renderSelect('rc_interpolation'           ,sysConfig.rc_interpolation, RC_INTERPOLATION);
```
Comment (mikeller) [inline js/header_dialog.js]: Indentation.

Comment (McGiverGim) [reply js/header_dialog.js]: I will fix it now.

**Comment by McGiverGim** [comment]
@mikeller fixed indentation as per review.

**Pull request merged**